tests/Tests.AzureAppConfiguration/Tests.AzureAppConfiguration.csproj

```diff
@@ -7,6 +7,7 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\build\AzureAppConfiguration.snk</AssemblyOriginatorKeyFile>
     <IsPackable>false</IsPackable>
+    <JsonSerializerIsReflectionEnabledByDefault>false</JsonSerializerIsReflectionEnabledByDefault>
```

Comment:
Oh that's interesting, I added it just as an automatic check for reflection. I didn't know it could prevent you from compiling though, does it mention why? Anyone could still test it locally using this or the `PublishAot` property in an example, but if it's an issue then I could remove it.